### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://DenisTsyba.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://DenisTsyba.github.io/layout_search-bar-airbnb/report/html_report/)

- [x] Icon implemented using background-image CSS property
- [x] Inputs are written inside of 'form' tag with correctly passed attributes
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,13 +4,32 @@
...
+    <form
+    class="search-form"
+    data-qa="big">
+      <div class="search-icon"></div>
+      <input
+      type="text"
+      class="search-form__input search-form__input--big"
+      placeholder="Try “Los Angeles”"
+      data-qa="keypress">
+    </form>
```
Comment (maxim2310) [inline src/index.html:22]: format next one in same way
```suggestion
    <form class="search-form" data-qa="big">
      <div class="search-icon"></div>
      <input
        type="text"
        class="search-form__input search-form__input--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
    </form>
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,91 @@
...
+  box-sizing: border-box;
+}
+
+body{
+  color: (red 0.24, green 0.31, blue 0.38, alpha 1);
+  font-family: Avenir,sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
```
Comment (maxim2310) [inline src/style.css:18]: its redundant
```suggestion

```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,91 @@
...
+  font-style: normal;
+  font-weight: 300;
+}
+
+*{
+  box-sizing: border-box;
+}
+
+body{
+  color: (red 0.24, green 0.31, blue 0.38, alpha 1);
```
Comment (maxim2310) [inline src/style.css:13]: use hex format instead this

File: src/style.css [hunk 4]
```diff
@@ -1 +1,91 @@
...
+  outline: none;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color: (red 0.24, green 0.31, blue 0.38, alpha 1);
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
```
Comment (maxim2310) [inline src/style.css:54]:
```suggestion

```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,91 @@
...
+  height: 42px;
+}
+
+.search-form__input{
+  width: 100%;
+  outline: none;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color: (red 0.24, green 0.31, blue 0.38, alpha 1);
```
Comment (maxim2310) [inline src/style.css:49]: same as above

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but lets make some improvement

File: src/index.html [hunk 6]
```diff
@@ -4,13 +4,30 @@
...
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form" data-qa="big">
+      <div class="search-icon"></div>
+      <input
+      type="text"
+      class="search-form__input search-form__input--big"
+      placeholder="Try “Los Angeles”"
+      data-qa="keypress"
+      >
```
Comment (anastasiiavorobiova) [inline src/index.html:20]:
```suggestion
      <input
        type="text"
        class="search-form__input search-form__input--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```

File: src/index.html [hunk 7]
```diff
@@ -4,13 +4,30 @@
...
+    </form>
+
+    <form class="search-form search-form--small" data-qa="small">
+      <div class="search-icon search-icon--small"></div>
+      <input
+      type="text"
+      class="search-form__input search-form__input--small"
+      placeholder="Try “Los Angeles”"
+      data-qa="keypress"
+      >
```
Comment (anastasiiavorobiova) [inline src/index.html:30]:
```suggestion
      <input
        type="text"
        class="search-form__input search-form__input--small"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your page

**Review by OleziO — APPROVED** [review]

**Pull request closed without merging**